src/main/resources/views/blocks/topic/data.ftl

```diff
@@ -105,6 +106,42 @@
                     </div>
                 </div>
             </li>
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle"
+                   href="#"
+                   role="button"
+                   data-toggle="dropdown"
+                   aria-haspopup="true"
+                   aria-expanded="false">
+                    <strong>After offset:</strong> ()
```

Comment:
Change to After only, it break responsive design 

![image](https://user-images.githubusercontent.com/2064609/67603029-2deaa780-f778-11e9-9884-85f908e90057.png)